DomainContent/Mexico/tequilaSpawnerServer.js

```diff
@@ -0,0 +1,113 @@
+//
+// tequilaSpawnerServer.js
+// 
+// Author: Rebecca Stankus on 05/02/18
+// Copyright High Fidelity 2018
+//
+// Licensed under the Apache 2.0 License
+// See accompanying license file or http://apache.org/
+//
+/* global Pointers */
+
+(function() {
+    var _this;
+
+    var SEARCH_RADIUS = 0.1;
+    var SPAWN_CHECK_MS = 10000;
+
+    function TequilaSpawner() {
+        _this = this;
+    }
+
+    TequilaSpawner.prototype = {
+        remotelyCallable: ['spawnIfNeeded', 'spawnNewTequilaShot'],
+        preload: function(entityID) {
+            print("good morning sunshine");
+            _this.entityID = entityID;
+            _this.position = Entities.getEntityProperties(_this.entityID, 'position').position;
+            Script.setInterval(function(){
+                print("interval");
+                _this.spawnIfNeeded();
+            }, SPAWN_CHECK_MS);
+        },
+
+        spawnIfNeeded: function() {
+            var needToSpawn = true;
+            Entities.findEntities(_this.position, SEARCH_RADIUS).forEach(function(element){
+                var name= Entities.getEntityProperties(element, 'name').name;
+                if (name === "Tequila Shot") {
+                    needToSpawn = false;
+                }
+            });
+            if (needToSpawn) {
+                _this.spawnNewTequilaShot();
+            }
+        },
+
+        createDrunkZone: function() {
```

Comment:
Is this the same zone that's created in the client?

Reply:
It's created in the server script, the client only checks to see if it's there in order to see if the user already has been drinking.